packages/xrpl/test/integration/transactions/nftokenMint.test.ts

```diff
@@ -0,0 +1,78 @@
+import { assert } from 'chai'
+import _ from 'lodash'
+
+import {
+  Client,
+  convertStringToHex,
+  getNFTokenID,
+  NFTokenMint,
+  TransactionMetadata,
+} from '../../../src'
+import serverUrl from '../serverUrl'
+import {
+  setupClient,
+  teardownClient,
+  XrplIntegrationTestContext,
+} from '../setup'
+
+// how long before each test case times out
+const TIMEOUT = 20000
+
+describe('NFTokenMint', function () {
+  let testContext: XrplIntegrationTestContext
+
+  beforeEach(async () => {
+    testContext = await setupClient(serverUrl)
+  })
+  afterEach(async () => teardownClient(testContext))
+
+  it(
+    'get NFTokenID',
+    async function () {
+      const client = new Client('wss://s.altnet.rippletest.net:51233/')
```

Comment:
You can just pass a different url to `setupClient` if you want and it also gives you a funded wallet.

Reply:
`setupClient` currently relies on owning the network in order to fund the account - it uses logic like the faucet to try and send money from the account that starts with all the money. Currently for this test though I have to use testnet because our standalone node doesn't support NFT transactions. (There's a separate ticket to update our standalone docker container)

So for now instead of updating `setupClient` to have 2 separate ways of creating a client & wallet, I'm just going to use `testnet` for this test. The real fix imo will be updating the docker container so that all transactions can use standalone mode normally :)

Reply:
@JST5000 sorry I don’t follow. It seems your code is equivalent to setupClient and setupClient accepts whatever network you want. Can you show me at the code level what’s different?

Reply:
Resolved offline :)